lib/logger.js

```diff
@@ -468,6 +487,9 @@ class Logger {
    */
 
   writeStream(level, module, args) {
+    if (this.truncating)
```

Comment:
This was necessary? New logs should be added, and then previous logs removed correct?

Reply:
It might not be necessary. I was trying to clear out issues where half of a line was stuck in the middle of the log, sometimes with the newline cutoff making the previous message long and weird. Especially with async `truncate`, I think it races against `writeStream`